### DISCUSSION TITLE
All old drum track code removed. And much more...

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1166] [Team Posts]
Hi. Hope you are all doing well.

MusE just lost a few significant pounds.

All 'old-style drum track' code was removed, and the 'DRUM' track type was removed,
 then 'NEW_DRUM' type was renamed 'DRUM'.
So we are now back to the original days of MusE with just a 'DRUM' track,
 except that it is now all the 'new-style' drum track code.

And several other streamlines, fixes, and features (mainly midnam controller support).
See ChangeLog.

**Comment by kybos** [comment]
Thanks, this is a big improvement! There was so much of the obsolete drum code everywhere...
